mathml/presentation-markup/radicals/root-parameters-2.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Radical parameters</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#radicals-msqrt-mroot">
+<meta name="assert" content="Test edge kerning values for radicals.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/mathml/support/feature-detection.js"></script>
+<style>
+  @font-face {
+      font-family: radical-negativekernbeforedegree1000-rulethickness1000;
+      src: url("/fonts/math/radical-negativekernbeforedegree1000-rulethickness1000.woff");
+  }
+  @font-face {
+      font-family: radical-kernafterdegreeminus5000-rulethickness1000;
+      src: url("/fonts/math/radical-kernafterdegreeminus5000-rulethickness1000.woff");
+  }
+  math, mspace {
+      font-size: 10px;
+  }
+  mspace {
+      opacity: .5;
+  }
+</style>
+<script>
+  var emToPx = 10 / 1000; // font-size: 10px, font.em = 1000
+  var epsilon = 1;
+
+  function getBox(aId) {
```

Comment:
Could maybe have a better name.

Reply:
i think we have several of these (it was copy and paste). maybe we should have a helper for this, but I don't know... I'll leave it as it for now.